### ISSUE TITLE
Validation dataset

### ISSUE THREAD

**Issue by AIRobotZhang** [cgnorthcutt/cleanlab#35]
Hello, if the confident learning just is applied in training dataset, the validation set keeps same to compare

**Comment by cgnorthcutt** [comment]
Hi, what is your issue? Please clarify.

**Comment by AIRobotZhang** [comment]
> Hi, what is your issue? Please clarify.

My issue is that whether only using Confident Learning to remove the error labelled data on training set or not. But the testing set still contains error labelled data.

**Comment by AIRobotZhang** [comment]
> > Hi, what is your issue? Please clarify.
> 
> My issue is that whether only using Confident Learning to remove the error labelled data on training set or not. But the testing set still contains error labelled data.

That's to say that the Test Dataset isn't removed error data by confident learning (CL).

**Comment by cgnorthcutt** [comment]
Use confident learning to remove error from any dataset you like... whether or not you 'call' the dataset test or train doesn't matter. Confident learning finds label errors in any dataset.

**Comment by AIRobotZhang** [comment]
> Use confident learning to remove error from any dataset you like... whether or not you 'call' the dataset test or train doesn't matter. Confident learning finds label errors in any dataset.

But how to show the Confident Learning is effective？Use the same test set before and after removing errors on training set for comparison？
![image](https://user-images.githubusercontent.com/20748608/86919995-9f7a5280-c15b-11ea-96bc-28a70e380d1d.png)
Whether or not the results of CL and baseline reported in the paper adopt the same test dataset.

**Comment by cgnorthcutt** [comment]
This is a question of how research is done and does not belong here. This forum is meant to discuss code issues with the cleanlab package. Ill answer this last question then close this thread. 

The table you are referring to uses a clean test set and a noisy train set. The baseline trains on the noisy train set then predicts on the clean test set. Confident Learning does the same, but finds the label errors, and trains on the remaining cleaned (originally noisy) training data after the label errors are removed. AFTER training, we measure the models prediction accuracy on a clean test set.

**Comment by AIRobotZhang** [comment]
> This is a question of how research is done and does not belong here. This forum is meant to discuss code issues with the cleanlab package. Ill answer this last question then close this thread.
> 
> The table you are referring to uses a clean test set and a noisy train set. The baseline trains on the noisy train set then predicts on the clean test set. Confident Learning does the same, but finds the label errors, and trains on the remaining cleaned (originally noisy) training data after the label errors are removed. AFTER training, we measure the models prediction accuracy on a clean test set.

I got it. Thank you for your patience to answer this question.
